### PR TITLE
Nightwatch Test for admin/content

### PR DESCRIPTION
**Pull request by porkloin**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/174

## Screenshot / UI changes

N/A

## Testing instructions

Run `yarn nightwatch` – test should wait for the select for content type to appear, click it, and ensure that "article", "page", and "recipe" are visible in the corresponding popover.

### CODE REVIEW

File: src/components/05_pages/Content/Content.js [hunk 1]
```diff
@@ -204,12 +204,13 @@ class Content extends Component {
               <Select
                 multiple
                 value={this.state.contentTypes}
+                data-nightwatch="content-type-select"
                 onChange={e => {
                   this.setState({ contentTypes: e.target.value }, () => {
                     this.props.requestContent(this.state);
                   });
                 }}
-                input={<Input id="select-multiple-checkbox" />}
```
Comment (dawehner) [inline src/components/05_pages/Content/Content.js]: I think we needed this ID for accessibility reasons, see the ```htmlFor``` property.

**Comment by lauriii** [comment]
Looks good! Thanks you! 🙌

File: src/tests/Nightwatch/Tests/AdminContent.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  '@tags': ['AdminContent'],
+  adminContentRenders(browser) {
+    browser
+      .logUserIn()
+      .relativeURL('/admin/content')
+      .waitForElementVisible('[data-nightwatch="content-type-select"]');
+
+    browser.click('[data-nightwatch="content-type-select"]');
```
[the remark this answers is not in the document]
Comment (justafish) [reply src/tests/Nightwatch/Tests/AdminContent.js]: That would be great, I'll merge this now and this can be done in a follow up :+1:

**Comment by justafish** [comment]
:+1: :+1: :+1:

**Pull request merged**